src/main/java/fun/epak/pak/controller/AuthController.java

```diff
@@ -20,8 +26,12 @@ public String getLogin() {
     }
 
     @PostMapping("/register")
-    public RedirectView postNewUser(UserRegistrationRequest userRegistrationRequest) {
-        userService.registerUser(userRegistrationRequest);
+    public RedirectView postNewUser(UserRegistrationRequest userRegistrationRequest,
+                                    @RequestParam("image") MultipartFile multipartFile) throws IOException {
```

Comment:
Nie chciałbym, żeby endpoint rzucał wyjątkiem. Żeby zrobić to zgodnie ze sztuką należałoby:
1. Złapać ten wyjątek w catchu i rzucić jakiś customowy rozszerzający RuntimeException np. `CouldNotStoreFileException`.
2. Stworzyć controller advice i złapać ten customowy wyjątek tam i zwrócić jakiś error widok. https://spring.io/blog/2013/11/01/exception-handling-in-spring-mvc